printer_mods/README.md

```diff
@@ -113,6 +113,7 @@ like so:
 |  | [Voron 2.4 cable clamps](./evandepol/Voron2.4-Cable-clamps) | A set of 2020-mounted cable guards to organize and dress the wiring on your build | :x: | :x: | :heavy_check_mark: |:x:|
 | falo | [Magnetic Exhaust Grill Cover](./falo/magnetic_grill_cover) | A magnetic cover to seal the filtered exhaust | :x: | :x: | :heavy_check_mark: | :x: |
 | FreeformFez | [V1.8 Filtered Power Inlet](./FreeformFez/FilteredPower) | V1.6 Filtered Inlet and Rocker Switch for V1.8 | :x: | :heavy_check_mark: | :x: |:x:|
+| GadgetAngel | [Cricut_Voron_Logos](./GadgetAngel/Cricut_Voron_Logos) |For any cutting machines or machines that relies on vector graphics and uses .svg files or vector files. These vectors or .svg files are the Voron Design Logos for the Voron 2.4, Voron SwitchWire and Voron 0.0/0.1. These .svg files use a minimal amount of anchor points to acheive the vector shapes! | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: |
```

Comment:
Please remove underscores inside the square brackets, that's the text shown to the user:

     [Cricut Voron Logos](./GadgetAngel/Cricut_Voron_Logos)

Replace `machines that relies on vector graphics and uses .svg files` with `machines that rely on vector graphics and use .svg files`.

Replace `acheive` with `achieve`

Reply:
done

Reply:
Did you push your changes?  I don't see any updates on the PR.

Reply:
Yes, all updates have now been pushed to the branch